confluent_kafka/schema_registry/schema.py

```diff
@@ -1,116 +1,204 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 import json
 
-from fastavro._schema import PRIMITIVES
-from fastavro.schema import schema_name, parse_schema
+PRIMITIVES = {'string', 'bytes', 'boolean', 'null',
+              'int', 'double', 'float', 'long'}
```

Comment:
This collection is really specific to Avro I think. It may seem general, but even if this type of collection were useful for another serialization type, there might be subtle differences, meaning it really is serialization type specific.